src/main/java/uk/gov/ch/oauth/Oauth2.java

```diff
@@ -30,27 +30,28 @@
 @Component
 public class Oauth2 implements IOauth {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger("docs.developer.ch.gov.uk");
+    private final Logger logger;
     private final IIdentityProvider identityProvider;
     private final SessionFactory sessionFactory;
     private final Duration timeoutDuration = Duration.ofSeconds(10L);
     private final NonceGenerator nonceGenerator = new NonceGenerator();
     private final OAuth2StateHandler oAuth2StateHandler;
     private static final String SIGN_IN_INFO = SessionKeys.SIGN_IN_INFO.getKey();
 
-
-
     @Autowired
     public Oauth2(final IIdentityProvider identityProvider, final SessionFactory sessionFactory) {
         this.identityProvider = identityProvider;
         this.sessionFactory = sessionFactory;
         oAuth2StateHandler = new OAuth2StateHandler(this.identityProvider);
+        logger = LoggerFactory.getLogger("docs.developer.ch.gov.uk");
```

Comment:
You'll remove this before committing to Oauth Library?

Reply:
Yeah or rename the var to oauth... if the OAuthCoordinator isn't in yet.

Reply:
Might want to speak with Rob about what his changes for Coordinator will look like.